source/guides/maintenance/cpu-performance.rst

```diff
@@ -193,6 +193,53 @@ group:
    After adding a new group, you must log out and log back in for the new group
    to take effect.
 
+Thermal configuration
+=====================
+
+Better thermal control and performance can be archived by providing platform
+specified configuration to thermald.
+
+`Linux DPTF Extract Utility`_ is a companion tool to thermald, This tool can
+make use of "Intel ® Dynamic Platform and Thermal Framework (`Intel DPTF`_)"
+technology, and convert to the thermal_conf.xml configuration format used
+by thermald.
+
+First, make sure thermald version is equal or above 1.9.1:
```

Comment:
```suggestion
First, make sure thermald version is equal to or above version 1.9.1:
```

Reply:
I'd rather make sure the version in Clear Linux is at least this version before publishing to avoid cluttering with static version check information. That sort of thing tends to be irrelevant quickly with the pace that Clear Linux updates move.

Reply:
Current version in Clear Linux is 1.9.
I submit a patch to upgrade to 1.9.1, and is under review now.